src/MongoDB.Driver.Core/MongoDB.Driver.Core.targets

```diff
@@ -7,4 +7,11 @@
             <Link>snappy64.dll</Link>
         </Content>
     </ItemGroup>
+
+    <ItemGroup Condition="$([MSBuild]::IsOsPlatform('Windows'))">
+        <Content Include="$(MSBuildThisFileDirectory)../x64/native/windows/libzstd.dll">
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Link>libzstd.dll</Link>
+        </Content>
+    </ItemGroup>
```

Comment:
I don't understand these packaging issues.

I defer to you and @vincentkam on them.